assets/src/js/admin/onboarding-wizard/components/select-input/index.js

```diff
@@ -4,15 +4,16 @@ import PropTypes from 'prop-types';
 // Import styles
 import './style.scss';
 
-const SelectInput = ( { value, onChange, options } ) => {
+const SelectInput = ( { label, value, onChange, options } ) => {
 	const optionElements = options.map( ( option, index ) => {
 		return (
 			<option value={ option.value } key={ index }>{ option.label }</option>
 		);
 	} );
 	return (
 		<div className="give-obw-select-input">
-			<select value={ value } className="give-obw-select-input" onChange={ ( event ) => onChange( event.target.value ) } >
+			{ label && ( <label className="give-obw-select-input__label">{ label }</label> ) }
```

Comment:
Looks like the `<label />` needs to either wrap the related `<input />` or be associated with a `for` attribute.

Reply:
Good catch, I've decided to go the `for` and `id` route, first converting the label to kebab case, then using it for these attributes. Ref: https://github.com/impress-org/givewp/commit/29d3294a150ea9345ae6760cb8b3bf38a0eee92e